src/main/java/gameframework/game/GameLevel.java

```diff
@@ -1,7 +1,14 @@
 package gameframework.game;
 
+/**
+ * Interface for game levels.
+ *
+ * Technically speaking, a "GameLevel" is just a thread whose initialize/simulate/free the level.
+ */
 public interface GameLevel extends Runnable {
+	/** Start the level (aka thread). */
```

Comment:
Can you explain when/in which conditions this can/should be used, and what will be the result?